### PR TITLE
feat(lpa-submissions-e2e-tests): add other policies page

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-17

## Description of change

add other policies page

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by a6software — APPROVED** [review]
LGTM

**Review by davehlewis — CHANGES_REQUESTED** [review]
There are also too few commits for this - should be 1 for tests, 1 for web app work, and 1 for reply service work. This is because although in the same repo they are discrete packages. At the very least web app and reply service need to be separate as it will mess with versioning (the reply service work here is a fix not a feat for example).

File: lpa-submissions-e2e-tests/cypress/integration/otherRelevantPolicies.feature [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      When Check your Answers is displayed
+      Then other relevant policies question heading is shown and the uploaded file name should be displayed
+
+    Scenario: AC15 Change answers
+      Given a change to answer 'Other relevant policies' is requested from Change your answers page
+      When an answer is saved
+      Then progress is made to the Check Your Answers page
+      And the updated answer is displayed
+
+  #   # Scenario: AC16 PDF & Horizon
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/integration/otherRelevantPolicies.feature:123]: Agreed. Removed.

File: lpa-submissions-e2e-tests/cypress/integration/otherRelevantPolicies/otherRelevantPolicies.step.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+When('other relevant policies question is requested', () => {
+  goToOtherRelevantPoliciesPage();
+});
+
+Then('LPA Planning Officer is presented with the ability to upload the policies', () => {
+  cy.verifyPage(page.url);
+  cy.verifyPageTitle(page.title);
+  cy.verifyPageHeading(page.heading);
+  cy.verifySectionName(page.section);
+  cy.checkPageA11y(`/${defaultPathId}/${page.url}`);
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/integration/otherRelevantPolicies/otherRelevantPolicies.step.js:50]: Agreed, removed.

**Review by davehlewis — APPROVED** [review]

**Pull request merged**